getting-started.md

```diff
@@ -7,7 +7,7 @@ Places requires special provisioning context with your regular Adobe Experience
 To get provisioned, complete the following tasks:
 
 * In the Admin Console, ensure that your organization is provisioned with Places.   If you are not the Adobe administrator, contact the administrator to ensure the Places context in Admin Console.
```

Comment:
for all of these bullets, please move the 2nd sentence 1 line down and line up the text with the bullet text.